products/idn/docs/identity-now/global-platform-resources/index.md

```diff
@@ -0,0 +1,21 @@
+---
+id: global-platform-resources
+title: Global Platform Resources
+pagination_label: Global Platform Resources
+sidebar_label: Global Platform Resources
+sidebar_position: 1
+sidebar_class_name: globalResources
+keywords: ['IdentityNow', 'resources', 'configuration']
+description: All the documentation that applies to the entire IdentityNow application
+slug: /docs/saas-connectivity/global-platform-resources
+tags: ['Connectivity']
+---
+
+These following pages show advanced setup and instructions on configuration in IdentityNow 
```

Comment:
Can you change this to "Developers can use global platform resources to improve all IdentityNow platforms. Read these documents for advanced setup and configuration instructions: "